package.json

```diff
@@ -9,7 +9,9 @@
   "scripts": {
     "start": "ember server",
     "build": "ember build",
-    "test": "ember test"
+    "client-test": "ember test",
```

Comment:
I think if we're going to bother running this for CI, we should maybe have it be `ember try:testall` as a smoke test.